public/maintenance.html

```diff
@@ -48,9 +48,17 @@
       <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
         <div class="govuk-grid-row">
           <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-l">Sorry, the service is unavailable</h1>
-            <p class="govuk-body">You will be able to use the service from 8:30pm.</p>
-            <p class="govuk-body">If you are part way through a check, we saved your answers. They will be available for 14 days from when you started the check.</p>
+            <h1 class="govuk-heading-l">Sorry, the service is unavailable because of planned maintenance</h1>
+            <p class="govuk-body">It will not be available on Wednesday 24 January 2024 from 6pm for up to 2 hours.</p>
+            <p class="govuk-body">Use our
+              <a href="https://www.gov.uk/guidance/funding-and-costs-assessment-for-civil-and-crime-matters">means assessment resources</a> for guidance on checking a client's eligibility.
```

Comment:
@willc-work Should we add `target="_blank"` within the html anchor tag so the links open up in a new tab?

Reply:
I didnt because if we wanted that to be the beahviour we would include that notice in the link and the docs dont show that,but I will confirm with Laurence

Reply:
Yeah that makes sense.